File: DAL/Models/DB/Category.cs
using System;
using System.Collections.Generic;

namespace DAL.Models.DB;

public partial class Category
{
    public int Id { get; set; }

    public string? Name { get; set; }

    public string? Description { get; set; }

    public string? Image { get; set; }

    public int? Status { get; set; }

    public DateTime? CreateAt { get; set; }

    public DateTime? UpdateAt { get; set; }

    public virtual ICollection<Recipe> Recipes { get; set; } = new List<Recipe>();
}

File: angular/src/app/models/category.ts
export class Category {
  id: number;
  name: string;
  description?: string;
  image?: string;
  status: number=1;
  countRecipe:number=0;

  constructor(id: number, name: string, description: string, image: string, status: number) {
    this.id = id;
    this.name = name;
    this.description = description;
    this.image = image;
    this.status = status;
  }

}
